<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandRatioDialog</class>
 <widget class="QDialog" name="BandRatioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>629</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Band Ratio Analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="minimumSize">
      <size>
       <width>707</width>
       <height>209</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>707</width>
       <height>209</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Spectrum Preview (At Spatial Origin)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCustomPlot" name="spectrumPlot" native="true">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>225</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Spectral Region of Interest</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Wavelength Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="rangeLabel">
        <property name="text">
         <string>200-2000 cm-1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Region 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="firstMinLineEdit">
          <property name="placeholderText">
           <string>Region 1 Left Bound</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstMaxLineEdit">
          <property name="placeholderText">
           <string>Region 1 Right Bound</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Region 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="secondMinLineEdit">
          <property name="placeholderText">
           <string>Region 2 Left Bound</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="secondMaxLineEdit">
          <property name="placeholderText">
           <string>Region 2 Right Bound</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="peakComboBox">
        <item>
         <property name="text">
          <string>Intensity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Area</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Area (Estimate Peak Edges)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="integrationComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Riemann Sum</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Peak Determination</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="integrationLabel">
        <property name="text">
         <string>Integration Method</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="formGroupBox_3">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Map Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="1">
       <widget class="QComboBox" name="gradientComboBox">
        <property name="toolTip">
         <string>An asterix by the color scheme name indicates a divergent color scheme particularly suited for this kind of map.</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>ColorBrewer BlueGreen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer BluePurple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer GreenBlue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer OrangeRed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer PurpleBlue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer PurpleBlueGreen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer PurpleRed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer RedPurple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer YellowGreen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer YellowGreenBlue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer YellowOrangeBrown</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer YellowOrangeRed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer Blues</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer Greens</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer Oranges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer Purples</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer Reds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ColorBrewer Grayscale</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Grayscale</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Night</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Candy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Ion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Thermal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ QCustomPlot Polar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ QCustomPlot Spectrum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Jet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Hues</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Hot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCustomPlot Cold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer BrownBlueGreen </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer PinkYellowGreen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer PurpleGreen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer PurpleOrange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer RedBlue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer RedGray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer RedYellowBlue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer RedYellowGreen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ ColorBrewer Spectral</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>↔ Vespucci Spectral</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Color Gradient</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="mapCheckBox">
        <property name="text">
         <string>Create Map</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BandRatioDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BandRatioDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
